protocols/index/test/Index-test.js

```diff
@@ -1,128 +0,0 @@
-
-  describe('Set', async () => {
-    it('Sets a locator for Alice', async () => {
-      await index.setLocator(aliceAddress, 2000, aliceAddress)
-    })
-
-    it('Sets a locator for Bob', async () => {
-      await index.setLocator(bobAddress, 500, bobAddress)
-    })
-
-    it('Sets a locator for Carol', async () => {
-      await index.setLocator(carolAddress, 1500, carolAddress)
-    })
-
-    it('Sets a locator for David', async () => {
-      await index.setLocator(davidAddress, 100, davidAddress)
-    })
-
-    it('Sets a locator of 0 for zara', async () => {
-      await index.setLocator(zaraAddress, 0, zaraAddress)
-    })
-
-    it("Sets a locator for Eve equal to Bob's locator", async () => {
-      await index.setLocator(eveAddress, 500, eveAddress)
-    })
-
-    it('Ensure ordering is correct', async () => {
-      const locators = await index.fetchLocators(7)
-      assert(locators[0] == aliceLocator, 'Alice should be first')
-      assert(locators[1] == carolLocator, 'Carol should be second')
-      assert(locators[2] == bobLocator, 'Bob should be third')
-      assert(locators[3] == eveLocator, 'Eve should be fourth')
-      assert(locators[4] == davidLocator, 'David should be fifth')
-      assert(locators[5] == zaraLocator, 'Zara should be last')
-    })
-  })
-
-  describe('Get', async () => {
-    it('Gets the locator for Alice', async () => {
-      equal((await index.getLocator(aliceAddress)).data, aliceLocator)
-    })
-
-    it('Gets the locator for Bob', async () => {
-      equal((await index.getLocator(bobAddress)).data, bobLocator)
-    })
-
-    it('Gets the locator for Carol', async () => {
-      equal((await index.getLocator(carolAddress)).data, carolLocator)
-    })
-
-    it('Gets the locator for David', async () => {
-      equal((await index.getLocator(davidAddress)).data, davidLocator)
-    })
-
-    it('Gets the locator for Eve', async () => {
-      equal((await index.getLocator(eveAddress)).data, eveLocator)
-    })
-
-    it('Gets the locator for Zara', async () => {
-      let zaraLocator = await index.getLocator(zaraAddress)
-      equal(zaraLocator.data, zaraLocator.data)
-      equal(zaraLocator.score, 0)
-    })
-
-    it('Gets a non existent locator', async () => {
-      let fredLocator = await index.getLocator(fredAddress)
-      equal(fredLocator.data, emptyLocator)
-    })
-  })
-
-  describe('Fetch', async () => {
-    it('Fetches locators', async () => {
-      const locators = await index.fetchLocators(7)
-      assert(locators[0] == aliceLocator, 'Alice should be first')
-      assert(locators[1] == carolLocator, 'Carol should be second')
-      assert(locators[2] == bobLocator, 'Bob should be third')
-      assert(locators[3] == eveLocator, 'Eve should be fourth')
-      assert(locators[4] == davidLocator, 'David should be fifth')
-      assert(locators[5] == zaraLocator, 'Zara should be last')
-      assert(BN(await index.length()).eq(6), 'Index length is incorrect')
-    })
-  })
-})
```

Comment:
https://github.com/airswap/airswap-protocols/commits/0117c18b2796f78d60ae5c90d020e044bb541aa8/protocols/index/test/Index-test.js

weird history.

Reply:
lol where did this come from

![bitmoji](https://render.bitstrips.com/v2/cpanel/33dd5e8f-cc39-4cc1-af24-e49c70b17dc9-cff72edb-7595-4ab6-8bc6-fd1729be0c15-v1.png?transparent=1&palette=1&width=246)